app/code/Magento/Eav/Model/Validator/Attribute/Code.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
```

Comment:
It's a good idea to use strict types within a scope of new implementations

Reply:
You have implemented the naming in the same manner as other validators. So, it's ok.
You could use an alias in the `use` section for this class. Something like:
```
use Magento\Eav\Model\Validator\Attribute\Code as AttributeCodeValidator
```

Reply:
@rogyar the alias was added.